apis/acceleratedmobilepageurl/v1.ts

```diff
@@ -16,7 +16,9 @@
 
 /* jshint maxlen: false */
 
-import {createAPIRequest} from '../../lib/apirequest';
+import {
```

Comment:
Why did it get formatted like that?  Guessing clang will just put it back after we run `gts fix` against it.

Reply:
want me to run `gts fix` on this PR?

Reply:
If you can - guessing that will be very rough

Reply:
if I have to do it 1 file at a time...yeah...no. Looks like the templates just need to be fixed.